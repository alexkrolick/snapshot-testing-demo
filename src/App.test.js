import React from "react";
import { create as render } from "react-test-renderer";
import App from "./App";

it("renders without crashing", () => {
  const component = render(<App />);
  const snap = component.toJSON();
  expect(snap).toMatchSnapshot();
});


