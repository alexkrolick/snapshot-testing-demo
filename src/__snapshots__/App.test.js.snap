// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="App"
>
  <div />
  <p
    className="App-intro"
  >
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
</div>
`;
